[2025-03-31T11:33:34.503+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:33:27.770896+00:00 [queued]>
[2025-03-31T11:33:34.522+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:33:27.770896+00:00 [queued]>
[2025-03-31T11:33:34.522+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-31T11:33:34.547+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): move_to_transformation> on 2025-03-31 10:33:27.770896+00:00
[2025-03-31T11:33:34.553+0100] {standard_task_runner.py:60} INFO - Started process 6228 to run task
[2025-03-31T11:33:34.560+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag2', 'move_to_transformation', 'manual__2025-03-31T10:33:27.770896+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/test_dag2.py', '--cfg-path', '/tmp/tmpp_lrhsvd']
[2025-03-31T11:33:34.564+0100] {standard_task_runner.py:88} INFO - Job 198: Subtask move_to_transformation
[2025-03-31T11:33:34.693+0100] {task_command.py:423} INFO - Running <TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:33:27.770896+00:00 [running]> on host LAPTOP-A4DHQTS9.
[2025-03-31T11:33:34.979+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag2' AIRFLOW_CTX_TASK_ID='move_to_transformation' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T10:33:27.770896+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T10:33:27.770896+00:00'
[2025-03-31T11:33:35.008+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-31T11:33:35.022+0100] {sql.py:450} INFO - Running statement: 
        SELECT column_name FROM information_schema.columns 
        WHERE table_schema = 'transformation' 
        AND table_name = 'trans_prog3'
        AND column_name <> 'uid';
    , parameters: None
[2025-03-31T11:33:35.035+0100] {sql.py:459} INFO - Rows affected: 3
[2025-03-31T11:33:35.052+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-31T11:33:35.070+0100] {sql.py:450} INFO - Running statement: 
        INSERT INTO transformation.trans_prog3
        SELECT * FROM staging.stg_prog3
        ON CONFLICT (uid) DO UPDATE 
        SET "lastupdated" = EXCLUDED."lastupdated", "Area (H)" = EXCLUDED."Area (H)", "farmers" = EXCLUDED."farmers"
        WHERE transformation.trans_prog3.lastupdated < EXCLUDED.lastupdated;
    , parameters: None
[2025-03-31T11:33:35.074+0100] {sql.py:459} INFO - Rows affected: 2
[2025-03-31T11:33:35.075+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-03-31T11:33:35.112+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_dag2, task_id=move_to_transformation, execution_date=20250331T103327, start_date=20250331T103334, end_date=20250331T103335
[2025-03-31T11:33:35.174+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-31T11:33:35.259+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
